src/app/pages/RewardPage/components/ClaimForms/FeesEarnedClaimRow/index.tsx

```diff
@@ -91,7 +96,7 @@ export const FeesEarnedClaimRow: React.FC<IFeesEarnedClaimRowProps> = ({
           textClassName="tw-text-xs tw-overflow-visible tw-font-bold"
           disabled={isClaimDisabled}
           title={
-            (claimFeesEarnedLocked &&
+            ((claimFeesEarnedLocked || assetClaimLocked) &&
               t(translations.maintenance.claimRewards).replace(
                 /<\/?\d+>/g,
```

Comment:
title is not being used in the ActionButton component.  as a result users can't see a reason for this button being disabled
<img width="228" alt="Screen Shot 2022-10-17 at 2 24 54 PM" src="https://user-images.githubusercontent.com/25097709/196160002-6b03f6c8-1a9e-4ee7-83ca-0e0162fe1f3f.png">

Reply:
as discussed, you have to mouseover  - title prop is applied via prop spreading in ActionButton